ddtrace/internal/context_manager.py

```diff
@@ -2,18 +2,20 @@
 import threading
 from ddtrace.vendor import six
 
+from ddtrace.compat import contextvars
 from .logger import get_logger
 from ..context import Context
 
 log = get_logger(__name__)
 
-try:
-    from contextvars import ContextVar
+_DD_CONTEXTVAR = contextvars.ContextVar("datadog_contextvar", default=None)
 
-    _DD_CONTEXTVAR = ContextVar("datadog_contextvar", default=None)
-    CONTEXTVARS_IS_AVAILABLE = True
+try:
+    import contextvars
```

Comment:
if we import from `compat` do we need this here?

Reply:
We still need to know whether the real contextvars are available or not for the reasons described below

Reply:
It probably makes more sense that the compat module provide the flag instead